packages/checkbox/checkbox.test.ts

```diff
@@ -226,13 +226,13 @@ describe('checkbox prompt', () => {
     events.keypress('space');
     expect(getScreen()).toMatchInlineSnapshot(`
       "? Select a number
+       ◯ 6
        ◯ 7
        ◯ 8
        ◯ 9
        ◯ 10
        ◯ 11
-      ❯◉ 12
-       ──────────────"
```

Comment:
Not sure if this is desired or not... This change means we'd never show bottom separator?